modules/responses.js

```diff
@@ -0,0 +1,47 @@
+
+
+exports.sendError = function(res) {
+	var error = {
+		status: 0,
```

Comment:
why we use status :0 and 1 here

Reply:
I will implement the status codes in this code . we did not learn about the status codes till now.